docs/continuous-delivery/gitops/secret-management/secret-management-gitops.md

```diff
@@ -0,0 +1,189 @@
+# Secret Management with GitOps
+
+GitOps projects have git repositories in the center that are considered a source of truth for managing both infrastructure and application. This infrastructure and application will require secured access to other resources of the system through secrets. Committing clear-text secrets into git repositories is unacceptable even if the repositories are private to your team and organization. This is unacceptable for a multitude of reasons, but an example can be because secrets are then stored in clear-text on every developers computer and if a developers computer is compromised, so are all the secrets. Teams need a secure way to handle secrets when using GitOps.
```

Comment:
```suggestion
GitOps projects have git repositories in the center that are considered a source of truth for managing both infrastructure and application. This infrastructure and application will require secured access to other resources of the system through secrets. Committing clear-text secrets into git repositories is unacceptable even if the repositories are private to your team and organization.  Teams need a secure way to handle secrets when using GitOps.
```

Reply:
I think we can skip the last sentence, to make it crisper, it should be obvious to everyone why we should not store secrets in repos.

Reply:
I am more than happy to do this, however you must know your comment directly conflicts with @John-Garland's on the same section 😄